docs/01 - build.md

```diff
@@ -109,17 +129,45 @@ If you want to reuse an existing Kubernetes config, such as your [`EKS`](https:/
 
-
-
-### Setup a Kubernetes development environment
-To run `KubeBlocks`, you needs `Docker` and a `Kubernetes` 1.24.1+ cluster for development.
-
-#### Docker environment
-1. Install [Docker](https://docs.docker.com/install/)
-   > For Linux, you'll have to configure docker to run without `sudo` for the KubeBlocks build scripts to work. Follow the instructions to [manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user).
-2. Create your [Docker Hub account](https://hub.docker.com/signup) if you don't already have one.
-
-#### Kubernetes environment
-- Kubernetes cluster
-  You can use cloud Kubernetes service, such as [`EKS`](https://aws.amazon.com/eks/) [`GKE`](https://cloud.google.com/kubernetes-engine) [`AKS`](https://azure.microsoft.com/en-us/products/kubernetes-service/), or use local Kubernetes cluster, such as [`Minikube`](https://minikube.sigs.k8s.io/docs/) [`k3d`](https://k3d.io/stable/).
-- For development purposes, you will also want to follow the optional steps to install [Helm 3.x](https://helm.sh/docs/intro/install/).
+### An example
+
+Here we present an example of deploying `KubeBlocks` locally(Bring your own toolbox) and utilizing it to create an `apeloud-mysql` cluster.
+
+Before you start, please ensure that you have prior experience with the [basic usage of KubeBlocks](https://kubeblocks.io/docs/preview/user_docs/overview/introduction). You can attempt to create a mysql cluster and establish a connection to it.
+
+#### Deploy KubeBlocks
+- Setup local cluster(Minikube, k3d), e.g., `k3d cluster create mycluster`
+- Install add-on. You can install add-ons through [Helm](https://kubeblocks.io/docs/release-0.8/developer_docs/integration/how-to-add-an-add-on). Here is a simpler example:
+    - Install [`kbcli`](https://kubeblocks.io/docs/release-0.8/user_docs/installation/install-with-kbcli/install-kbcli)
+    - Install [`KubeBlocks`](https://kubeblocks.io/docs/release-0.8/user_docs/installation/install-with-kbcli/install-kubeblocks-with-kbcli): `kbcli kubeblocks install`. This will install `KubeBlocks` and start running its manager.
+    - Install add-ons you need. Please follow guildlines [here](https://kubeblocks.io/docs/preview/user_docs/overview/supported-addons#list-addons). 
+```shell
+# make sure there is an index
+kbcli addon index list
+# search supported addon
+kbcli addon search apecloud-mysql  
+# install addon
+kbcli addon install apecloud-mysql --index kubeblocks --version [YOUR_VERSION]
+# enable addon
+kbcli addon enable apecloud-mysql
+# list the addons again to check whether it is enabled.
+kbcli addon list
+```
+- To deploy KubeBlocks of your version locally, stop the manager of `KubeBlocks` you just install: 
+```shell
+kubectl scale deployment kubeblocks --replicas=0 -n kb-system
+```
```

Comment:
These lines are more suitable for a separate section titled such as "How to Deploy Your Development (e.g., Debugging) Version of the Controller."

Reply:
Otherwise, if a reader execute this command, the subsequent commands will not be able to run.